terminus-ui/paginator/src/paginator.component.spec.ts

```diff
@@ -1,470 +1,309 @@
+import { Type } from '@angular/core';
 import {
-  SimpleChange,
-  SimpleChanges,
-} from '@angular/core';
-import { ChangeDetectorRefMock } from '@terminus/ngx-tools/testing';
+  ComponentFixture, fakeAsync, tick,
```

Comment:
Let's drop these items onto their own lines. Our linter doesn't quite get it for us yet :/